action.yml

```diff
@@ -0,0 +1,41 @@
+name: 'DevCycle Test Harness'
+description: 'Run test harness tests against given SDKs'
+inputs:
+  sdk:
```

Comment:
technically this is meant to potentially be a list of SDKs to test, which would make this `sdks`

also might as well just use the same name as the env var to prevent confusion, `sdks-to-test`

Reply:
fixed